doc/classes/MultiMeshInstance.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MultiMeshInstance" inherits="GeometryInstance" category="Core" version="3.0-rc1">
 	<brief_description>
-		Node that instances a [MultiMesh].
+		Node that Instances a [MultiMesh].
 	</brief_description>
 	<description>
-		MultiMeshInstance is a [Node] that takes a [MultiMesh] resource and adds it to the current scenario by creating an instance of it (yes, this is an instance of instances).
+		MultiMeshInstance is a [Node] that takes a [MultiMesh] resource and adds it to the current scenario by creating an instance of it (yes, this is an instance of instances).  This is usually used to speed up rendering of continuous/high number of mesh of same virtual object like trees and grasses.  Note :- not all graphics cards support,  multi mesh rendering. 
```

Comment:
Formatting can be made better. There are some double-spaces (after the periods), a hanging hyphen (`-`) after the colon (`:`), and a trailing space after the last period.
Also, "continuous/high number" would be better as just "high number". "Virtual" might be dropped as well, unless I'm missing something. "number of mesh" should be "number of meshes" or "amount of the same mesh".
Finally, there is an extra comma after "support".